src/app/organization/tellers/create-teller/create-teller.component.ts

```diff
@@ -0,0 +1,88 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DatePipe } from '@angular/common';
+
+/** Custom Services */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/**
+ * Create Create Teller component.
+ */
+@Component({
+  selector: 'mifosx-create-teller',
+  templateUrl: './create-teller.component.html',
+  styleUrls: ['./create-teller.component.scss']
+})
+export class CreateTellerComponent implements OnInit {
+
+  /** Teller Form */
+  tellerForm: FormGroup;
+  /** Office data. */
+  officeData: any;
+   /** Minimum Date allowed. */
+   minDate = new Date(2000, 0, 1);
+   /** Maximum Date allowed. */
+   maxDate = new Date();
+
+   /**
+    * Retrieves the offices data from `resolve`.
+    * @param {FormBuilder} formBuilder Form Builder.
+    * @param {OrganizationService} organizationService Organization Service.
+    * @param {ActivatedRoute} route Activated Route.
+    * @param {Router} router Router for navigation.
+    * @param {DatePipe} datePipe Date Pipe to format date.
+    */
+  constructor(private formBuilder: FormBuilder,
+    private organizationService: OrganizationService,
```

Comment:
```suggestion
            private organizationService: OrganizationService,
```
Ensure proper indentation so that all of the parameters passed to the constructor are aligned.